packages/components/header/_header.scss

```diff
@@ -80,427 +51,251 @@
       content: "";
     }
   }
-
-  &:hover,
-  &:active,
-  &:focus {
-    background-color: transparent;
-  }
 }
 
-.nhsuk-header__logo--only {
-  max-width: 100%;
-
-  @include mq($from: tablet) {
-    .nhsuk-header__link--service {
-      align-items: center;
-      display: flex;
-      -ms-flex-align: center;
-      margin-bottom: 0;
-      width: auto;
-    }
+.nhsuk-header__link,
+.nhsuk-header__transactional-service-name--link {
+  @include nhsuk-font(19, $line-height: 1.1);
+  @include nhsuk-link-style-header;
+  text-decoration: none;
+  text-wrap: balance;
 
-    .nhsuk-header__service-name {
-      padding-left: nhsuk-spacing(3);
-    }
+  &:hover {
+    text-decoration: underline;
   }
 }
 
-.nhsuk-header__content {
-  @include print-hide();
-  position: relative;
+.nhsuk-header__service-name {
+  margin-bottom: -(nhsuk-spacing(2));
+  margin-top: -(nhsuk-spacing(2));
+}
 
-  &.js-show {
-    border-bottom: nhsuk-spacing(1) solid $color_nhsuk-grey-5;
-  }
+.nhsuk-header__transactional-service-name {
+  align-items: center;
+  display: flex;
+  margin-bottom: -(nhsuk-spacing(2));
+  margin-top: -(nhsuk-spacing(2));
+}
 
+/**
+ * Content area (top-right)
+ */
+.nhsuk-header__content {
   @include mq($from: tablet) {
-    &.js-show {
-      border-bottom: 0;
-    }
+    margin-left: auto;
   }
 }
 
+/**
+ * Search
+ */
 .nhsuk-header__search {
+  @include print-hide();
   position: relative;
-  text-align: right;
-
-  @include mq($from: tablet) {
-    margin-left: nhsuk-spacing(2);
-  }
+  z-index: 13;
 }
 
 .nhsuk-header__search-form {
+  display: flex;
   height: 100%;
   overflow: visible;
 
   @include mq($until: tablet) {
-    display: flex;
-    margin: nhsuk-spacing(3) 0 0;
     position: relative;
-    width: 100%;
-  }
-}
-
-.nhsuk-header__search-wrap {
-  @include mq($from: tablet) {
-    display: block;
   }
 }
 
 .nhsuk-search__input {
-  -webkit-appearance: listbox;
-  border-bottom-left-radius: $nhsuk-border-radius;
-  border-bottom-right-radius: 0;
-  border-top-left-radius: $nhsuk-border-radius;
-  border-top-right-radius: 0;
+  border: 1px solid $color_nhsuk-white;
+  border-radius: $nhsuk-border-radius 0 0 $nhsuk-border-radius;
+  font-size: $nhsuk-base-font-size;
+  height: 40px;
   padding: 0 nhsuk-spacing(3);
+  width: 100%;
```

Comment:
Suggest moving this into the media query below, currently it's squashing the button to 40px wide on over tablet size.

Reply:
Done.